a2wsgi/asgi.py

```diff
@@ -174,7 +175,7 @@ def _done_callback(future: asyncio.Future) -> None:
         )
         run_asgi.add_done_callback(_done_callback)
 
-        read_count, body = 0, environ["wsgi.input"]
+        read_count, body = 0, environ.get("wsgi.input", StringIO())
```

Comment:
Should use `BytesIO`.

In PEP3333.

```
An input stream (file-like object) from which the HTTP request body bytes can be read. (The server or gateway may perform reads on-demand as requested by the application, or it may pre- read the client’s request body and buffer it in-memory or on disk, or use any other technique for providing such an input stream, according to its preference.)
```

Reply:
In #26, we use `not environ["wsgi.input"]`. Maybe we should use `environ["wsgi.input"] or BytesIO()` to set empty stream at here.